.ipynb_checkpoints/README-checkpoint.md

```diff
@@ -0,0 +1,48 @@
+# Chromatic Effects on PSF and Shear Measurement for Roman
```

Comment:
I put a whole load of comments here, but I just realized this is a checkpoint file?  Why is the readme in here?  My comments still apply but they should be used to change the README.md in the repo, not this file